code/modules/mob/living/living.dm

```diff
@@ -1487,10 +1488,25 @@
 		if(WABBAJACK_MONKEY)
 			new_mob = new /mob/living/carbon/human/species/monkey(loc)
 
+		if(WABBAJACK_CLOWN)
+			var/picked_clown = pick(typesof(/mob/living/basic/clown))
+			new_mob = new picked_clown(loc)
+
 		if(WABBAJACK_ROBOT)
 			var/static/list/robot_options = list(
 				/mob/living/silicon/robot = 200,
 				/mob/living/basic/drone/polymorphed = 200,
+				/mob/living/basic/bot/dedbot = 50,
+				/mob/living/basic/bot/cleanbot = 50,
+				/mob/living/basic/bot/firebot = 50,
+				/mob/living/basic/bot/honkbot = 50,
+				/mob/living/basic/bot/hygienebot = 50,
+				/mob/living/basic/bot/medbot/mysterious = 50,
+				/mob/living/basic/bot/medbot = 50,
+				/mob/living/basic/bot/vibebot = 50,
```

Comment:
```suggestion
				/mob/living/basic/bot/dedbot = 25,
				/mob/living/basic/bot/cleanbot = 25,
				/mob/living/basic/bot/firebot = 25,
				/mob/living/basic/bot/honkbot = 25,
				/mob/living/basic/bot/hygienebot = 25,
				/mob/living/basic/bot/medbot/mysterious = 12,
				/mob/living/basic/bot/medbot = 13,
				/mob/living/basic/bot/vibebot = 25,
```
I think the sum of the weights of these bots shouldn't be greater than the top two picks.